src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -6,7 +6,7 @@
 import AdmZip from 'adm-zip'
 import os from 'os'
 import fs from 'fs-extra'
-import parseDiff from 'parse-diff'
```

Comment:
I will formally remove "parse-diff" from package.json / package-lock.json in a separate PR